ui/src/main/js/page/user/user/UserCopyModal.js

```diff
@@ -0,0 +1,176 @@
+import React, { useEffect, useState } from 'react';
+import { createUseStyles } from 'react-jss';
+import { useDispatch, useSelector } from 'react-redux';
+import { saveUser, validateUser } from 'store/actions/users';
+import { fetchRoles } from 'store/actions/roles';
+import Modal from 'common/component/modal/Modal';
+import DynamicSelectInput from 'common/component/input/DynamicSelectInput';
+import PasswordInput from 'common/component/input/PasswordInput';
+import TextInput from 'common/component/input/TextInput';
+import * as HTTPErrorUtils from 'common/util/httpErrorUtilities';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import UserEnum from 'common/util/enums/User';
+
+const useStyles = createUseStyles({
+    descriptorContainer: {
+        display: 'flex',
+        alignItems: 'center', 
+        padding: [0, 0, '20px', '60px']
+    },
+    descriptor: {
+        fontSize: '14px',
+        paddingLeft: '8px'
+    }
+})
+
+const UserCopyModal = ({ data, isOpen, toggleModal, copiedUsername }) => {
```

Comment:
Can we add prop types for validation to this component?  And defaults if possible.